src/main/java/mate.academy/lib/Inject.java

```diff
@@ -1,5 +1,12 @@
 package mate.academy.lib;
 
-public @interface Inject {
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
 
+@Retention(RetentionPolicy.RUNTIME)
+@Target(ElementType.FIELD)
+public @interface Inject {
```

Comment:
The `@Inject` annotation is correctly defined with the required `@Retention` and `@Target`. Good job adhering to the checklist.